index.js

```diff
@@ -1,9 +1,33 @@
 'use strict';
-const urlRegex = require('url-regex');
+const urlRegex = require('url-regex')();
```

Comment:
It returns a method for a reason. You should not initiate it here.

Reply:
done